tmessage/cli.py

```diff
@@ -35,6 +37,22 @@ def on_message(client, userdata, message):
     if user != current_user:
         print(Back.GREEN + Fore.BLACK + current_msg +
               Back.RESET + Fore.RESET + "")
+        store_messages(current_msg)

```

Comment:
Message received will be stored with a different username (the one who send the message)